docs/new_horizen/03-proof_submission_interface/02-rpc_methods.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Proof Submission Interface RPC
+---
+
+# Proof Submission Interface Methods
+Below are the details about the API interface exposed by the Mainchain node to Proof Submitters.  
+
+## SettlementFflonkPallet
+- **Type:** non-blocking
+- **Description:** submit the proof and return the transaction hash if it is valid, Error otherwise.
+```
+Signature: submitProof(proof, public_inputs) -> Result<tx_hash, Error>
+```
+
+In case of success, the proof is included in the mempool, relayed, and eventually included in a proposed block and then finalized.
+
+**Note:** each different proof verification pallet will expose this API.
+
+## RPC Commands
+- **Endpoint:** PoE (Proof of Existence)
+- **Description:** get the Merkle path of the proof within the provided attestation, returning Error if the attestation_id’s Attestation event isn’t published yet.
+```
+Signature: proofPath(attestation_id, proof_leaf) -> Result<merkle_path, Error> Type: non-blocking
+```
+
+For the initial launch, the management of the calls to these APIs is “embedded” into the PolygonCDK core code and is not available out-of-the-box to the end user.
```

Comment:
```suggestion
Currently these APIs are “embedded” into the PolygonCDK core code and not available the end user.
```

Reply:
I don't think we mention the PolygonCDK anywhere (except in the diagram). Not sure we want to keep this line.